src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -370,6 +371,20 @@ func run(prg string, args ...string) (string, error) {
 	return string(out), nil
 }
 
+func base64Encode(sites []string) string {
+	raw, err := json.Marshal(sites)
+	if err != nil {
+		log.Errorf("Unable to marshal proxied sites: %s", err)
```

Comment:
This should not happen, maybe a panic would be more useful here? it would alert the developer who is building the proxied sites list immediately.